### DISCUSSION TITLE
Welcome to machinaris Discussions!

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#102] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chrslcy** [comment]
Hi,

Would someone please share how to install Machinaris on Ubuntu? Sorry, but I'm a complete novice to Ubuntu and Git. 

Chia, Plotman and MadMax are already installed on the machine.

Many thanks,

Chris

**Reply by olivierLA75** [reply]
Actually looking for the same, would appreciate any help :) thanks!

**Reply by chrslcy** [reply]
Wow. I totally missed this on the main page:

"To get started with Machinaris, follow an install guide for your platform: <a href="https://github.com/guydavis/machinaris/wiki/Windows">Windows</a>, <a href="https://github.com/guydavis/machinaris/wiki/Linux">Linux</a>, <a href="https://github.com/guydavis/machinaris/wiki/MacOS">Macintosh</a>, <a href="https://github.com/guydavis/machinaris/wiki/Unraid">Unraid</a>, and <a href="https://github.com/guydavis/machinaris/wiki/Docker">others</a>."

**Reply by guydavis** [reply]
Also, the primary benefit of Machinaris is the all-in-one install using a single Docker run command for Chia, Plotman, Chiadog, Madmax, Flax, and Flaxdog.  You don't need to install this other software separately.  Machinaris bundles it all as per the [main page](https://github.com/guydavis/machinaris):

![image](https://user-images.githubusercontent.com/12936/124514631-b8cbd400-dd9a-11eb-9730-16a7a499f5b6.png)
